### DISCUSSION TITLE
User feedback

### DISCUSSION THREAD

**Discussion by felixendres** [linuxdeepin/developer-center#3423] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hi,

first of all thanks for putting your software out there as open source. I am using deepin-camera 1.0.0.16-1 from the Arch Linux (Manjaro) package.

It is very good, but there are a few points that make me stick with an alternative (cheese), and I just wanted to let you know about it:
- The "Start recording" and "Stop recording" buttons are not in the same place, when I take a video. So I cannot just click the mouse button, but must move the cursor again to find the Stop button.
- I can open the folder with the videos, but I cannot open the video directly

Things I liked:
- Easy to switch from taking photo to video
- Shows last pictures/videos as thumbnails
- Big buttons
- Can set the output resolution and location

Best regards

**Comment by BLumia** [comment]
Hey thanks for the feedback. Hi, sorry for the late reply since we didn't check issues that were reported outside the developer-center repo. I'll help you move the issue there and see if we can add such features to the project :)

**Reply by BLumia** [reply]
cc @meiyixiang
